### ISSUE TITLE
Abstract Out To Functions

### ISSUE THREAD

**Issue by Ayamin0x539** [Catastrio/Twitch-Follower-Grabber#23]
Right now, your script is just a script. I'd like to see more modularity. Namely, create functions

```python
def get_user_by_id(userId):
    # ...
```
and
```python
def get_user_by_username(username):
    # ...
```

That abstracts the functionality of what you already have implemented. 
Then you can just make function calls.
Furthermore, someone *else* can import your script as a module and use the functionality as well.

Finally, make a function
```python
def get_followers_for_username(username):
    # ...
```

Which is the bulk of your entire script. It should be simple.

It might help to make a function 
```python
def convert_id_to_username(idList):
    # ...
```
as well, to take a list of IDs and turn them into a list of usernames via API calls (what you're doing in the script).

Basically, turn your script into 4 functions. Take it to the next level. Support other developers with these convenience methods. Create the bridge between Python code and twitch API calls.

**Comment by Catastrio** [comment]
It's been a while since I've done things with functions.

Just to clarify, I'm going to make 4 functions, then have a main() script that calls on those functions correct?

So it'll still function entirely the same, but I'll have functions I can either recycle for future programs or other developers can make use of by importing into programs of their own right?

**Comment by Ayamin0x539** [comment]
Right. You could use one more function though. See other issues.
